dids/src/test/kotlin/web5/sdk/dids/DidIonTest.kt

```diff
@@ -41,6 +44,19 @@ class DidIonTest {
     assertTrue(did.creationMetadata!!.longFormDid.startsWith(did.uri))
   }
 
+  @Test
+  @Ignore("For demonstration purposes only - relies on network and AWS configuration")
```

Comment:
cool, eventually we should add a lot of these in another demonstration file maybe

Reply:
Definitely! I tried to do a bit of that in the `examples.md` file, but wanted to have a test file to at least ensure the syntax is valid.